src/components/Footer/TodoFooter.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-param-reassign */
+import { useAppContextContainer } from '../../context/AppContext';
+import { footerOption } from './TodoFooter.data';
+import TodoFooterOption from './TodoFooterOption';
+
+const TodoFooter = () => {
+  const { changeFilterType, filterType, todos, deleteCompleted } =
+    useAppContextContainer();
+
+  const completed = todos?.reduce((a, b) => (!b.completed ? (a += 1) : a), 0);
```

Comment:
This variable name is bad on first sight i thought it counts the completed todos.